packages/arb-token-bridge-ui/src/pages/index.tsx

```diff
@@ -16,5 +19,18 @@ const App = dynamic(() => import('../components/App/App'), {
 })
 
 export default function Index() {
+  useEffect(() => {
+    // user-added custom chains do not persists between sessions
+    // we add locally stored custom chains
```

Comment:
Why can't this hook be implemented as a `Syncer` component and placed inside the App just like we have other things hydrate of initialization?

Reply:
I need to do this right on index.tsx initialization, if I add it as a syncer inside App.tsx it won't work - the custom network is added too late and doesn't get recognized